content/content-strategy/cms/cms-requirements/section2.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: text/textblock
+---
+Hosting and security is vital to any CMS, especially in government. This is an important part of choosing the right CMS. Things like where your website content is hosted and what levels of security are in place, should be considered as part of the choosing process.
+### [1]Service level agreement
+It is also valuable to ask about a ‘service level agreement’ from your CMS provider. This is the contract that defines the level of service expected from the service provider. It defines what the customer will receive. 
+### [1]Security
```

Comment:
Incorrect display level. Should be ### [2] to make a heading 3 appear correctly.

Reply:
Steve. Fixed.